lua/telescope/utils.lua

```diff
@@ -170,6 +170,81 @@ utils.filter_symbols = function(results, opts, post_filter)
   end
 end
 
+utils.path_filename_first = (function()
+  return function(path, path_display)
```

Comment:
why is this not a normal function? why a function that returns a function and gets executed immediatly

Reply:
Done ✅ (pushed)